src/main/resources/org/jenkinsci/plugins/oic/OicSecurityRealm/help-logoutFromOpenidProvider.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Logout from Jenkins entails logout from OpenID Provider.
+    Logout from Jenkins entails logout from OpenID Provider. Please note that this will make all API Keys invalid.
```

Comment:
I understand why this is, but it may be problematic for some users who would expect that their API token should still function as a valid means of access even if they have logged out from the OpenID provider

Reply:
This not the intent ; the logout here applies when the user logout from Jenkins.
User cannot reasonably assume the token should still work.

It is rather the other way around, when the user logs out of Jenkins, we make the logout from OP optional because:

> https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout
> At the Logout Endpoint, the OP SHOULD ask the End-User whether to log out of the OP as well. [...]

But some OP logs out the user from of all RPs.

Reply:
Wait. Are we talking about jenkins API token ?

Reply:
yes, jenkins api tokens - sorry for the confusion

Reply:
This is a necessary stalemate. See https://github.com/jenkinsci/oic-auth-plugin/pull/310#discussion_r1638780197 for details and reasoning. If you have something else in mind, please do let me know. cc: @jglick

Reply:
nope @krezovic , i have come to the same conclusion (thanks for your patience!)